qiskit/visualization/pulse/matplotlib.py

```diff
@@ -507,8 +607,28 @@ def _draw_labels(self, ax, labels, prev_labels, dt, y0):
                 ax.axvline(tf*dt, -1, 1, color=color,
                            linestyle=linestyle, alpha=alpha)
 
-    def _draw_channels(self, ax, output_channels, interp_method, t0, tf, dt, scale_dict,
-                       label=False, framechange=True):
+    def _draw_channels(self, ax: 'matplotlib.axes.Axes',
+                       output_channels: Dict['Channel', EventsOutputChannels],
+                       interp_method: Callable,
+                       t0: int, tf: int, dt: float,
+                       scale_dict: Dict['Chanenl', float],
+                       label: bool = False, framechange: bool = True) -> float:
+        """Draw pulse instructions on given mpl axis.
+
+        Args:
+            ax: axis object to draw pulses.
+            output_channels: Event table of channels.
+            interp_method: Callback function for waveform interpolation.
+            t0: Start time of schedule.
+            tf: End time of schedule.
+            dt: Time interval.
+            scale_dict: Scale factor for each channel.
+            label: When set `True` draw labels.
+            framechange: When set `True` draw frame change symbols.
+
+        Return:
+            y0: Value of final vertical axis of canvas.
```

Comment:
```suggestion
            Value of final vertical axis of canvas.
```
The name of the local variable used within the function and later returned doesn't have any meaning to the caller